WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -398,11 +406,11 @@ public static boolean postExists(long blogId, long postId) {
     }
 
     /*
-     * returns true if any posts in the passed list exist in this list
+     * returns true if any posts in the passed list exist in this list for the given tag
      */
-    public static boolean hasOverlap(ReaderPostList posts) {
+    public static boolean hasOverlap(ReaderPostList posts, ReaderTag tag) {
         for (ReaderPost post : posts) {
-            if (postExists(post.blogId, post.postId)) {
+            if (ReaderPostTable.postExistsForReaderTag(post.blogId, post.postId, tag)) {
```

Comment:
```suggestion
            if (postExistsForReaderTag(post.blogId, post.postId, tag)) {
```

We can remove the `ReaderPostTable.` prefix for this method call.  If you prefer it for clarity, just let me know.

Reply:
I personally prefer having the prefix for static methods, but I don't mind it either way, so if you'd like we can make the change.

Reply:
If that's your preference, we can leave it.  It's not a problem at all.

Reply:
It looks like everywhere else we are using the methods without the prefix, so let's make the change to keep things consistent. Thanks for bringing this up.